Makefile

```diff
@@ -392,15 +392,11 @@ test-integration-docker-logs: ## Runs Docker Logs integration tests
 
 .PHONY: test-integration-elasticsearch
 test-integration-elasticsearch: ## Runs Elasticsearch integration tests
-ifeq ($(AUTOSPAWN), true)
-	@scripts/setup_integration_env.sh elasticsearch stop
-	@scripts/setup_integration_env.sh elasticsearch start
-	sleep 60 # Many services are very slow... Give them a sec...
-endif
-	${MAYBE_ENVIRONMENT_EXEC} cargo test --no-fail-fast --no-default-features --features es-integration-tests --lib ::elasticsearch::
-ifeq ($(AUTODESPAWN), true)
-	@scripts/setup_integration_env.sh elasticsearch stop
-endif
+	RUST_VERSION=${RUST_VERSION} ${CONTAINER_TOOL}-compose -f scripts/integration/docker-compose.elasticsearch.yml run --rm runner
+
+.PHONY: test-integration-elasticsearch-cleanup
+test-integration-elasticsearch-cleanup:
+	${CONTAINER_TOOL}-compose -f scripts/integration/docker-compose.elasticsearch.yml rm -fsv
```

Comment:
Wonder if it's worth using the long option names  for easier readability

Reply:
I've not preference 🤷  I just used the long version to be consistent with other services...

Reply:
You mean used the _short_ version, yeS?

Reply:
oh, I misunderstood what you meant 😅

Reply:
I'll open a PR to have a generic `make` command like this

```
test-integration-%-cleanup:
    things -f docker-compose.$*.yml otherthings
```

So I'll refactor it at this time 😉

Reply:
and add the `AUTODESPAWN` also

Reply:
😆 No worries. I've a bad habit of not doing it either, but it's definitely nice to have the long options when you look back at the functions months later thinking "what does this do again???"